src/Uno.UWP/WinRT/MarshalString.skia.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation.Metadata;
+
+namespace Windows.WinRT;
+
+internal class MarshalString
+{
+	public unsafe static string FromAbi(IntPtr value)
+	{
+		if (value == IntPtr.Zero)
+		{
+			return "";
+		}
+		uint length = default(uint);
+		char* value2 = Platform.WindowsGetStringRawBuffer(value, &length);
+		return new string(value2, 0, (int)length);
+	}
+
+	public static void DisposeAbi(IntPtr hstring)
+	{
+		if (hstring != IntPtr.Zero)
+		{
+			_ = Platform.WindowsDeleteString(hstring);
+		}
+	}
+
+	private static class Platform
+	{
+		[DllImport("api-ms-win-core-winrt-string-l1-1-0.dll", CallingConvention = CallingConvention.StdCall)]
```

Comment:
Will this fail gracefully when running on older versions of Windows?

Reply:
this is likely to fail fatally unless you are on windows 10 or later
any recommendation on how to recover from?

Reply:
given the fact that we only support minimum of .net7 (soon 8) or higher now
and starting from net7, win7 and win8.1 are no longer supported
i think this is a non-issue

Reply:
@jeromelaban @MartinZikmund what are your opinions?

Reply:
Windows 7, 8 and a few other Windows 10 are not supported anymore, and people are having increasingly difficult time to get it running there (https://github.com/dotnet/core/issues/7556#issuecomment-1786377918). 

If we find someone broken by this, we'll adjust if we can.